src/exporters/qemu.rs

```diff
@@ -80,11 +80,11 @@ impl QemuExporter {
                             }
                         }
                         let tdiff = time_tdiff.total_time_jiffies()
-                            * procfs::ticks_per_second().unwrap() as f32;
+                            * procfs::ticks_per_second().unwrap() as u64;
```

Comment:
Despite the name, the old `total_time_jiffies` function was actually returning the total number of seconds.  This value was then multiplied by `ticks_per_second` to get the total number of ticks.

But now with the procfs upgrade, the `total_time_jiffies` function is directly returning the total number of ticks, so you don't need to multiply the result by `ticks_per_second`.

Reply:
Thanks !